docs/setup-cluster/deploy-cluster/slurm/singularity.rst

```diff
@@ -182,3 +184,40 @@ You can view the current set of Docker image names in the cache with the ``-l``
       manage-singularity-cache -l
       determinedai/environments:py-3.8-pytorch-1.10-tf-2.8-cpu-096d730
       determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-096d730
+
+.. _manage-enroot-cache:
+
+**********************************************************************
+ Managing the Enroot Image Cache using the manage-enroot-cache script
+**********************************************************************
+
+This script, ``/usr/bin/manage-enroot-cache``, simplifies the management of a set of shared Enroot
+.sqsh file downloads and then creates an Enroot container for use by the current user. It provides
+the following features:
+
+   -  Download the Determined default cuda, cpu, or rocm environment images
```

Comment:
Okay, I followed the "Managing the Singularity Image Cache using the manage-singularity-cache script", I will fix both

Reply:
It might be helpful to talk about -s further as this is actual documentation as opposed to just CLI help message which is necessarily not huge.   Maybe something like this:


When using `manage-enroot-cache` you must provide a temporary directory via the `-s` option which is used to download (enroot import) the associated enroot .sqsh file.   The .sqsh file is read by the `enroot create` command to generate the container.   The directory need only be accessible on the local host.   If the directory you specify is shared with other users, the script will re-use any downloaded .sqsh files and directly `enroot create` an enroot container without needing a separate download.